### PR TITLE
Use UTF-8 rather than the platform-dependant default charset

### PR DESCRIPTION
**Pull request by omnivion**
### Information
- JIRA story QPPCT-569.

### Changes proposed in this PR:
- Ensure we use UTF-8 across the project at runtime

### Checklist 
- [X] All JUnit tests pass (`mvn clean verify`).
- [X] New unit tests written to cover new functionality.
- [X] Added and updated JavaDocs for non-test classes and methods.
- [X] No local design debt. Do you feel that something is "ugly" after your changes?
- [X] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 import java.io.UncheckedIOException;
 import java.nio.charset.Charset;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java]: MINOR Remove this unused import 'java.nio.charset.Charset'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.nio.charset.Charset;
```
Comment (FlexionSonarQube) [inline rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java]: MINOR Remove this unused import 'java.nio.charset.Charset'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sfradkin** [comment]
@omnivion: Please remove the imports and then this can be merged.

File: converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 package gov.cms.qpp.conversion.xml;
 
-import java.io.UncheckedIOException;
-import java.nio.charset.Charset;
-import org.jdom2.Element;
-import org.jdom2.JDOMException;
-import org.jdom2.input.SAXBuilder;
-
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
```
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/xml/XmlUtils.java]: MINOR Remove this unused import 'java.nio.charset.Charset'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java [hunk 4]
```diff
@@ -22,9 +17,15 @@
...
-import java.io.IOException;
-import java.nio.charset.Charset;
+import gov.cms.qpp.conversion.Converter;
+import gov.cms.qpp.conversion.api.model.Constants;
+import gov.cms.qpp.conversion.api.model.ErrorMessage;
+import gov.cms.qpp.conversion.correlation.PathCorrelator;
+import gov.cms.qpp.conversion.encode.JsonWrapper;
+import gov.cms.qpp.conversion.model.error.AllErrors;
+import gov.cms.qpp.conversion.model.error.Error;
+import gov.cms.qpp.conversion.model.error.QppValidationException;
```
Comment (FlexionSonarQube) [inline rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java]: MINOR Remove this unused import 'java.nio.charset.Charset'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Pull request merged**